platform/retarget.cpp

```diff
@@ -389,6 +441,12 @@ extern "C" int _fstat(int fd, struct stat *st) {
         st->st_mode = S_IFCHR;
         return  0;
     }
+#if defined MBED_CFG_DEBUG_OPTIONS_COVERAGE
+    else if (coverage_report && fd == gcov_fd){
+        st->st_size = 0;
+        return 0;
+    }
+#endif
```

Comment:
Nit: Same deal in the above block about spaces, there should be spaces after the `if ()`.

Ex. `if (blah){` should be `if (blah) {`.